tests/test_backends.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from django.core import mail
+
+
+@pytest.fixture
+def email_backend(settings):
+    """
+    Overrides the EMAIL_BACKEND setting, because "pytest-django" uses
+    "django.test.utils.setup_test_environment", where  it is set to
+    "django.core.mail.backends.locmem.EmailBackend".
+    """
+    settings.EMAIL_BACKEND = 'django_postmark_utils.backends.EmailBackend'
+
+
+@pytest.fixture
+def postmark_response(postmark_request):
+    """
+    Mocks an error response from a Postmark API call via Postmarker.
+    """
+    postmark_request.return_value.json.side_effect = ValueError
```

Comment:
Shouldn't we be mimicking a Postmarker exception being thrown, rather than a `ValueError`?